services/console-proxy/server/pom.xml

```diff
@@ -49,6 +49,16 @@
       <artifactId>cloudstack-service-console-proxy-rdpclient</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-server</artifactId>
+      <version>9.2.20.v20161216</version>
```

Comment:
Use jetty version as defined in root pom.xml, instead of using hard-coded versions use a `cs.xxx` config/variable